bson/src/main/org/bson/codecs/pojo/ConventionSetPrivateFieldImpl.java

```diff
@@ -49,6 +52,12 @@ public void apply(final ClassModelBuilder<?> classModelBuilder) {
 
         private PrivateProperyAccessor(final PropertyAccessorImpl<T> wrapped) {
             this.wrapped = wrapped;
+            try {
+                wrapped.getPropertyMetadata().getField().setAccessible(true);
```

Comment:
Doing it here means it doesn't need to be done in the set method.

Reply:
Sorry must have ctrl-z'd that back